src/PageContent/Calendar/Dialog.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import './Dialog.css';
+import Header from './Header';
+import DialogContent from './DialogContent';
+
+let values = {
```

Comment:
See [new comment](https://github.com/huangjoyce3/MealMatch/pull/27#discussion_r180329598) in #27. Add this dummy data into the `delivery` object in the following format (inside `delivery`:
```
delivererGroup: {
    name: 'Green Greeks',
    deliverers: [
        {
             name: 'Blake Johnson',
             phone: '206-876-5432',
             email: 'blake@greengreeks.org'
        }
        {
             name: 'Erika Zhang',
             phone: '206-876-5432',
             email: 'erika@greengreeks.org'
        }
    ]
}
```